app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -304,6 +328,7 @@ public Survey closeSurvey(GuidCreatedOnVersionHolder keys) {
         Survey existing = getSurvey(keys);
         existing.setPublished(false);
         existing.setModifiedOn(DateUtils.getCurrentMillisFromEpoch());
+        existing.setSchemaRevision(null);
```

Comment:
A survey that is unpublished may still be referenced in a cache implemented by a client. Is it okay if that survey has no schema associated with it when the user submits answers to that old and potentially unpublished version of the survey? We still accept those answers.